client/player.js

```diff
@@ -26,10 +27,21 @@ var Player = function (position_, world_) {
     name: "scout",
     reloadTime:100,
     loadStatus:100,
-    magazine:300,
+    magazine:50,
     maxMagazine:300,
   };
 
+  // These are the starting values for each type. Players can gather more shots up to the limit for that type.
```

Comment:
client should not be responsible for keeping track of this. the server should manage this

Reply:
Moved to server.